CHANGELOG.md

```diff
@@ -5,7 +5,6 @@
 
 **Implemented enhancements:**
 
-- Do not allow hyphenated cookbook names to be generated [\#915](https://github.com/chef/chef-dk/pull/915)
```

Comment:
Do you mean, leave these messages in and add a new entry for switching to a warning?